android/app/src/main/java/com/reactnativenavigation/views/RnnToolBar.java

```diff
@@ -214,17 +214,16 @@ private void hideToolbar() {
     }
 
     public void setNavUpButton() {
-        setNavUpButton(null);
+        setNavUpButton(null, false);
     }
 
     @SuppressWarnings({"ConstantConditions"})
-    public void setNavUpButton(Screen screen) {
+    public void setNavUpButton(Screen screen, boolean isBack) {
```

Comment:
I took a look and that all seems good @guyca.  I probably won't have time to work on this until tomorrow, but I'll see what I can do to move this along.